draft-ietf-tls-ticketrequests.md

```diff
@@ -47,32 +47,33 @@ normative:
 --- abstract
 
 TLS session tickets enable stateless connection resumption for clients without
-server-side per-client state. Servers vend session tickets to clients, at their
-discretion, upon connection establishment. Clients store and use tickets when
-resuming future connections. Moreover, clients should use tickets at most once for
-session resumption, especially if such keying material protects early application
-data. Single-use tickets bound the number of parallel connections a client
-may initiate by the number of tickets received from a given server. To address
-this limitation, this document describes a mechanism by which clients may specify
-the desired number of tickets needed for future connections.
+server-side, per-client state. Servers vend an arbitrary number of session tickets
+to clients, at their discretion, upon connection establishment. Clients store and
+use tickets when resuming future connections. This document describes a mechanism by
+which clients may specify the desired number of tickets needed for future connections.
+This extension aims to remove server ambiguity surrounding ticket count while simultaneously
+reducing ticket waste and priming clients for future connection attempts.
 
 --- middle
 
 # Introduction
 
-As per {{RFC5077}}, and as described in {{RFC8446}},
-TLS servers send clients session tickets at their own discretion in NewSessionTicket messages.
-Clients are in complete control of how many tickets they may use when establishing
-future and subsequent connections. For example, clients may open multiple TLS connections to the same server
-for HTTP, or may race TLS connections across different network interfaces.
-The latter is especially useful in transport systems that implement Happy Eyeballs {{RFC8305}}.
-Since connection concurrency and resumption is controlled by clients, a standard mechanism
-to request more than one ticket is desirable.
-
-This document specifies a new TLS extension -- ticket_request -- that may be used
+As per {{RFC5077}}, and as described in {{RFC8446}}, TLS servers send clients an arbitrary
+number of session tickets at their own discretion in NewSessionTicket messages. There are
+two limitations with this design. First, servers typically guess (or hard-code) the number
```

Comment:
In what way do the servers guess? Do you mean that they guess the number that the client will need to use?

Reply:
They guess the number of tickets to vend.

Reply:
Guess implies that there is a correct answer. This text doesn't clearly define whose correct answer that is. The server just does what it wants for itself, it doesn't really guess. If it is guessing at all, it's guessing what the client would want, right?

Reply:
How about `servers typically determine some (often hard-coded) number of tickets...`

Reply:
I like it! I went with a variant: "servers choose some (often hard-coded) number.."